released/packages/coq-cfml-stdlib/coq-cfml-stdlib.20211215/opam

```diff
@@ -0,0 +1,41 @@
+
+opam-version: "2.0"
+maintainer: "arthur.chargueraud@inria.fr"
+
+homepage: "https://gitlab.inria.fr/charguer/cfml2"
+dev-repo: "git+https://gitlab.inria.fr/charguer/cfml2.git"
+bug-reports: "https://gitlab.inria.fr/charguer/cfml2/-/issues"
+license: "MIT"
+
+synopsis: "The CFML program verification system"
+description: """
+CFML is a program verification system for OCaml.
+"""
+
+build: [make "-C" "lib/stdlib" "-j%{jobs}%"]
+install: [make "-C" "lib/stdlib" "install"]
+depends: [
+  "coq" { >= "8.13" }
+  "cfml" { = version }
```

Comment:
Where is this package defined? Is it really upstream in OCaml?

Reply:
Hi Karl, the `cfml` package should appear in the OCaml package repository soon, I have made a PR for it. Perhaps I should have waited for it to appear before submitting this PR, sorry.

Reply:
Also, I should have submitted `coq-cfml-stdlib` and `coq-cfml-basis` in the same PR, sorry. (I will do so next time.)

Reply:
Sounds good, maybe you can do a force-push to to the PR once the `cfml` package has been merged. In the meantime, we can at least merge `coq-cfml-basis`.